requirements/edx/base.txt

```diff
@@ -93,7 +93,7 @@ djangorestframework-xml==1.4.0  # via edx-enterprise
 djangorestframework==3.9.4
 docopt==0.6.2
 docutils==0.16            # via botocore
-drf-yasg==1.17.0          # via edx-api-doc-tools
+drf-yasg==1.17.1          # via edx-api-doc-tools
```

Comment:
This version drops support for Python 3.5, we should probably constrain it for now: https://drf-yasg.readthedocs.io/en/stable/changelog.html

Reply:
Still need to run `make upgrade` to propagate the new constraint on this.

Reply:
still running.